### DISCUSSION TITLE
[Question][Hw4] Using mutate and update_seeds

### DISCUSSION THREAD

**Discussion by adilb99** [prosyslab-classroom/cs348-information-security#579] [Q&A]
Do we have to use `mutate` and `update_seeds` or can we make our own functions to use in `run` that are not necessarily called the same way?

Or do we absolutely have to use `mutate` and `update_seeds` and with the same parameters provided in the skeleton code?

**Comment by Junseon00** [comment]
It means that you didn't implement the `mutate` and `update_seed` function?
It's fine to define a new function, or change the types or etc... but we recommend you to follow our skeleton code.
